privacyidea/api/lib/postpolicy.py

```diff
@@ -262,18 +262,17 @@ def check_tokeninfo(request, response):
     :return: A new modified response
     """
     content = json.loads(response.data)
-    policy_object = g.policy_object
     serial = content.get("detail", {}).get("serial")
-    if serial:
+    tokeninfos_pol = g.policy_object.get_action_values(
+        ACTION.TOKENINFO,
+        scope=SCOPE.AUTHZ,
+        client=g.client_ip,
+        allow_white_space_in_action=True,
+        audit_data=g.audit_object.audit_data)
+    if serial and tokeninfos_pol:
         tokens = get_tokens(serial=serial)
```

Comment:
Yes, we could do the get_tokens *after* the get_action_values, but imho all in a block
~~~~
if serial:
~~~~

Reply:
will do!

Reply:
fixed in a1740172ecbba64493d89bb14edbd21e2f7e08f4